internal/osbuild/qemu_assembler.go

```diff
@@ -22,6 +22,7 @@ type QEMUPartition struct {
 	Size       uint64         `json:"size,omitempty"`
 	Type       *uuid.UUID     `json:"type,omitempty"`
 	Bootable   bool           `json:"bootable,omitempty"`
+	PARTUUID   *uuid.UUID     `json:"uuid,omitempty"`
```

Comment:
I think I prefer naming this field just `UUID` for the sake of consistency.

Reply:
I don't really have a strong opinion here. The field few lines above is called "PTUUID", so I thought it would be consistent like this.

Reply:
Yeah, but the `PTUUID` field is consistent with its json representation, whereas the `PTUUID` is not.

Reply:
ok, I'll change it